src/globals.js

```diff
@@ -4,15 +4,21 @@
  * consumers to decide. This makes dependency management a lot easier but comes
  * with some risks which can be allieviated with unit tests.
  */
+import lodash from 'lodash';
+import bluebird from 'bluebird';
```

Comment:
Making this optional missed the point of having it: not coupling this to any specific implementation of APIs which are ubiquitous throughout js eco system.

Also it's currently broken as lodash is only a dev dependency.

I think this should be reverted and the configure be a required step in the main file of applications. I did post it in the slack channel for feedback at the time to see if I was being crazy. I think this solves a very annoying issue for dependency management by just passing responsibility to the hopefully-intelligent consumer. I don't think it needs a layman option; this is software engineering 🌉